tests/unittests/Foundation/TestUtils.h

```diff
@@ -15,6 +15,17 @@
 //******************************************************************************
 #pragma once
 
-#import <Foundation/NSOrderedSet.h>
+#import <Foundation/Foundation.h>
+#import <functional>
+#include <memory>
 
-void assertOrderedSetContent(NSOrderedSet* set, NSObject* first, ...);
\ No newline at end of file
+#define _CONCAT(x, y) x##y
+#define CONCAT(x, y) _CONCAT(x, y)
+
+#define _SCOPE_GUARD(STATEMENT) std::unique_ptr<void, std::function<void(void*)>> CONCAT(_closeScope_, __LINE__)((void*)0x1, STATEMENT)
+
+void assertOrderedSetContent(NSOrderedSet* set, NSObject* first, ...);
+NSString* getModulePath();
```

Comment:
nothing specific, just the idea of 'moving these to a shared location' in general

---
In reply to: [73608630](https://github.com/Microsoft/WinObjC/pull/805#discussion_r73608630) [](ancestors = 73608630,73606151)